crates/turborepo-lib/src/run/summary/execution.rs

```diff
@@ -82,6 +86,12 @@ impl<'a> ExecutionSummary<'a> {
             String::new()
         };
 
+        let cache_line = if self.cached_remote > 0 {
+            format!("{} (of which {} remote)", self.cached, self.cached_remote)
```

Comment:
Don't want to bikeshed this a ton, but I feel like we can simplify this to just `10 (8 remote)` or `10 (2 local, 8 remote)`.